frappe/public/js/form_builder/components/Section.vue

```diff
@@ -24,7 +24,7 @@
 			>
 				<div class="section-label">
 					<EditableInput
-						:text="section.df.label"
+						:text="__(section.df.label)"
```

Comment:
I believe this is a small "mistake" sadly.

I'm a French developer, so my UI language in Frappe is in _French_, but I work on doctypes by giving them _English labels_ that I later translate (as usual). Sadly, translating the `EditableInput`'s value makes it harder for me to type my labels in English when my UI language.

Also I can't type labels starting with `B` because they are immediately transformed into `Mds` (_billions_) :laughing:

Should I just switch to English to work on DocTypes? I can't really expect my advanced users to do the same, so I believe they **_could_** break their labels by mistake in the Customize Form view.

Reply:
We can revert this part IMO.

Reply:
Rest of the PR is good yes :+1:, only the `<EditableInput :text>` prop is problematic.